### PR TITLE
Test against Python 3.4.

### PR DESCRIPTION
**Pull request by pope1ni**
As discussed in #264.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py{27,35,36}-django-111
```
Comment (mbaechtold) [inline tox.ini]: According to https://docs.djangoproject.com/en/1.11/faq/install/#what-python-version-can-i-use-with-django, Django 1.11 should run with Python 3.4 too. Could you please add it too?

Comment (camilonova) [inline tox.ini]: Please add here 34 as well

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ sudo: false
 language: python
 cache: pip
 python:
+- 2.7
```
Comment (mbaechtold) [inline .travis.yml]: Good catch, thank you!

File: tox.ini (hunk 1 above)
Comment (pope1ni) [reply tox.ini]: Yes. Quite right!

Comment (pope1ni) [reply tox.ini]: I think you both got there at the same time!

**Comment by pope1ni** [comment]
This should be all good pending the green tick...

**Comment by pope1ni** [comment]
So this looks good, as the tests pass, but we haven't restored anything such as use of `six.u()` (although use of `@python_2_unicode_compatible` from `django.utils.encoding` would be better.)

It may be that we can just get away without this which will make life easier.

@camilonova: Would it be possible to release v3.0.3 with the restored support if you're happy with this?

**Comment by camilonova** [comment]
Sure.

**Pull request merged**